python/paddle/fluid/dygraph/dygraph_to_static/break_continue_transformer.py

```diff
@@ -103,41 +104,22 @@ class BreakContinueTransformer(gast.NodeTransformer):
         set continue to False at the beginning of each loop
 
         TODO: more details should be summarized as design document
+
+    Note: The class is inherited from BaseNodeVisitor instead of NodeTransformer,
```

Comment:
As I commented below that "BaseNodeVisitor" had better to be "BaseNodeTransformer", you probably would like to change the Note here.

Reply:
As discuss offline